### PR TITLE
transfer annotations from string to list<string>

### PR DESCRIPTION
**Pull request by vasilich6107**
This PR converts `ClassProperty` annotation item to list.

This is a part of work towards fixing #41 #116

### CODE REVIEW

File: lib/generator.dart [hunk 1]
```diff
@@ -293,7 +293,7 @@ Make sure your query is correct and your schema is updated.''');
   }
 
   // On custom scalars
-  String annotation;
+  var annotations = <String>[];
```
Comment (comigor) [inline lib/generator.dart:296]: we can make it final

File: lib/generator.dart [hunk 2]
```diff
@@ -467,11 +470,12 @@ class _GeneratorVisitor extends RecursiveVisitor {
     final dartTypeStr = gql.buildTypeString(node.type, context.options,
         dartType: true, replaceLeafWith: nextClassName, schema: context.schema);
 
-    String annotation;
+    var annotations = <String>[];
```
Comment (comigor) [inline lib/generator.dart:473]: same

File: lib/generator.dart (hunk 1 above)
Comment (vasilich6107) [reply lib/generator.dart]: Done

File: lib/generator.dart (hunk 2 above)
Comment (vasilich6107) [reply lib/generator.dart]: Done

**Pull request merged**